public/content/roadmap/fusaka/index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Fulu-Osaka (Fusaka)
+description: Learn about the Fusaka protocol upgrade
+lang: en
+---
+
+# Fusaka {#fusaka}
+
+The Fusaka network upgrade follows [Pectra](/roadmap/pectra/) and brings more new features and improves the experience for every Ethereum user and developer. The name consists of execution layer upgrade Osaka and consensus layer version named after the Fulu star. Both parts of Ethereum receive an upgrade that push Ethereum scaling, security and user experience to the future. 
+
+This upgrade is planned for Q4 2025.
+
+<InfoBanner>
+The Fusaka upgrade is only a single step in Ethereum's long-term development goals. Learn more about [the protocol roadmap](/roadmap/) and [previous upgrades](/history/).
+</InfoBanner>
+
+## Improvements in Fusaka {#new-improvements}
+
+### Data availability & L2 scaling {#da-scaling}
+
+#### PeerDAS {#7594}
+
+Specification: https://eips.ethereum.org/EIPS/eip-7594
+
+Resources: https://youtu.be/bONWd1x2TjQ?t=328 (dapplion on PeerDAS)
+
+This is the *headliner* of the Fusaka fork, the main feature added in this upgrade. L2s currently post their data to Ethereum in blobs, the ephemeral data type created specifically for L2s. Pre-Fusaka, every full node has to store every **byte of those blobs to ensure that the data exists. As blob throughput rises, having to download all of this data would be untenably resource-intensive.
+
+With [Data Availability Sampling](https://notes.ethereum.org/@fradamt/das-fork-choice) , instead of having to download all of the blob data, each node will sample just a portion. Blobs are uniformly randomly distributed across nodes in the network with each full node holding only 1/8th of the data, therefore enabling theoretical scale up to 8x. To ensure availability of the data,  Any portion of the data can be reconstructed from any existing 50% of the whole with methods that drive down the probability of wrong or missing data to a cryptographically negligible level (~one in 10²⁰ to one in 10²⁴).
+
+This keeps hardware and bandwidth requirements for nodes tenable while enabling blob scaling resulting in more scale with smaller fees for L2s.
+
+In-depth: https://eprint.iacr.org/2024/1362.pdf
+
+#### Blob parameter only forks
+
+Specification: https://eips.ethereum.org/EIPS/eip-7892
+
+L2s scale Ethereum - as their networks grow, they need to post more data to Ethereum. This means that Ethereum will need to increase the number of blobs available to them as time goes on. Although PeerDAS enables scaling blob data, it needs to be done in gradually and safely. 
+
+Because Ethereum is code running on thousands of independent nodes that requirement agreement on the rules, you can’t just make changes like blob increases the way you deploy a website update. Any rule change must be a coordinated upgrade where every validator software upgrades at the same predetermined block.
```

Comment:
```suggestion
Because Ethereum is code running on thousands of independent nodes that require agreement on same rules, we cannot simply introduces changes like increasing blob count the way you deploy a website update. Any rule change must be a coordinated upgrade where every node, client and validator software upgrades before the same predetermined block.
```